### PR TITLE
Fix nex 1860/extract pdf assets on export

### PR DESCRIPTION
**Pull request by ktsirkou**
Related to [NEX-1860](https://oat-sa.atlassian.net/browse/NEX-1860)

Inserting media file from ckeditor `Insert Media` is interpreted as `<object>` element on qit.xml. 
During export of test/item we try to traverse the qti file and extract all asset.  Extracting assets from <object> elements currently doesn't include the `application/pdf` mime-type and the resulted export qti.xml to include the URL of the `taomedia://`  for the pdf asset.

This fix adds the pdf mime-type on the available exportable asset types from qti object element

### CODE REVIEW

**Review by peetya — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Code is self documented

**Review by wazelin — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

File: manifest.php [hunk 1]
```diff
@@ -43,7 +43,7 @@
     'name'        => 'taoQtiItem',
     'label'       => 'QTI item model',
     'license'     => 'GPL-2.0',
-    'version'     => '25.10.2',
+    'version'     => '25.10.3',
     'author'      => 'Open Assessment Technologies',
     'requires' => [
         'taoItems' => '>=10.11.0',
```
Comment (wazelin) [inline manifest.php:49]: Not really required, but can be bumped.
```suggestion
        'taoItems' => '>=10.13.0',
```

**Review by alroniks — APPROVED** [review]
I agree with @wazelin about version bump, but it is not strictly required.

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is `develop`

**Pull request merged**